app/config/scheme.js

```diff
@@ -142,6 +142,17 @@ const schema = Joi.object({
       inbound: Joi.string().required()
     }).required(),
     enabled: Joi.boolean().default(true)
+  }).required(),
+  sfiExpanded: Joi.object({
+    name: Joi.string().default(SFI_EXPANDED),
+    fileMasks: Joi.object({
+      inbound: Joi.array().items(Joi.string()).default([/^CTL_ESFIO\d{4}_AP_\d*.dat$/, /^CTL_ESFIO\d{4}_AP_\d*.txt$/])
+    }),
+    server: Joi.string().default(MANAGED_GATEWAY),
+    directories: Joi.object({
+      inbound: Joi.string().required()
+    }).required(),
+    enabled: Joi.boolean().default(true)
   }).required()
 })
```

Comment:
since I am doing all the validation messaging on other services, it would be good to add messages for JOI validation objects to create more meaningful error messages.

Reply:
When we do our config validations we throw an error, as so: 

![image](https://github.com/DEFRA/ffc-pay-gateway/assets/60177449/19d9d7d4-3dbc-489d-8394-e0cc7ee93c99)

Appreciate that's not as robust as what you're doing, but as these are specifically isolated to application configuration, wonder if this is acceptable in this context?

I'm mindful that this kind of logging is very prominent in Payments and introducing validations to this level is going to be a pretty huge exercise.

Reply:
I have inspected how the code works, and I understand that we already have a way to display error messages, if you are happy with default messages for debugging purposes, then it should be fine. However my concern lies within anyone else inspecting these errors, would they get enough context from default error messages to be able to understand what is wrong?(for example someone who oversees prod env and validation fails on default pattern .default([/^CTL_ESFIO\d{4}_AP_\d*.dat$/, /^CTL_ESFIO\d{4}_AP_\d*.txt$/] , the code that you have right now, wont display pattern that array's strings are expecting.)  If you think this should not be a problem I am happy to approve

Reply:
Shouldn't be a problem currently, although I agree there is scope for a continuous improvement ticket in this area